Cargo.toml

```diff
@@ -1,18 +1,17 @@
 [package]
 name = "linkerd2-mock-dst"
 version = "0.1.0"
-authors = ["Eliza Weisman <eliza@buoyant.io>"]
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
 edition = "2018"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+publish = false
 
 [dependencies]
-tokio = { version = "0.2", features = ["full", "parking_lot"] }
-tonic = "0.1.1"
-linkerd2-proxy-api-tonic = { git = "https://github.com/linkerd/linkerd2-proxy-api", branch = "eliza/tonic"}
+tokio = { version = "0.2", features = ["macros", "rt-threaded", "sync"] }
+tonic = "0.2.1"
+linkerd2-proxy-api = { git = "https://github.com/linkerd/linkerd2-proxy-api", features = ["transport"] }
```

Comment:
Should this be pinned to a SHA?

Reply:
i think it's okay for now, since the sha is pinned in the lockfile. we'll tag a release soon.